libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorAdaptiveRecognizer.cs

```diff
@@ -0,0 +1,334 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Schema;
+using Microsoft.BotFramework.Orchestrator;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.AI.Orchestrator
+{
+    /// <summary>
+    /// Class that represents an adaptive Orchestrator recognizer.
+    /// </summary>
+    public class OrchestratorAdaptiveRecognizer : Recognizer
+    {
+        /// <summary>
+        /// The Kind name for this recognizer.
+        /// </summary>
+        [JsonProperty("$kind")]
+        public const string Kind = "Microsoft.OrchestratorRecognizer";
+
+        /// <summary>
+        /// Property key in RecognizerResult that holds the full recognition result from Orchestrator core.
+        /// </summary>
+        public const string ResultProperty = "result";
+
+        private const float UnknownIntentFilterScore = 0.4F;
+        private static BotFramework.Orchestrator.Orchestrator orchestrator = null;
+        private string _modelPath;
+        private string _snapshotPath;
+        private ILabelResolver _resolver = null;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OrchestratorAdaptiveRecognizer"/> class.
+        /// </summary>
+        /// <param name="callerLine">caller line.</param>
```

Comment:
>caller [](start = 37, length = 6)

Initial capital on this and the next comment.

Reply:
Fixed.